src/golang/lib/context/context.go

```diff
@@ -76,10 +76,10 @@ func ParseAqContext(ctx context.Context) (*AqContext, int, error) {
 	storageConfig := config.Storage()
 
 	return &AqContext{
-		User: user.User{
-			Id:             userId,
-			OrganizationId: organizationId,
-			Auth0Id:        auth0Id,
+		User: models.User{
```

Comment:
Update "Id"s to "ID"s. Perhaps we can do that in a different PR.

Reply:
Yup we can do as much as possible in these PRs but here is the task for a catch all to do a post cleanup https://linear.app/aqueducthq/issue/ENG-2077/clean-up-codebase-naming-conventions-for-id-db-dag-etc